airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/ops/slack.py

```diff
@@ -18,4 +21,17 @@ def send_slack_webhook(webhook_url, report):
     webhook = WebhookClient(webhook_url)
     for msg in chunk_messages(report):
         # Wrap in code block as slack does not support markdown in webhooks
-        webhook.send(text=f"```{msg}```")
+        webhook.send(f"```{msg}```")
+
+
+def send_slack_message(context: OpExecutionContext, channel: str, message: str):
+    """
+    Send a slack message to the given channel.
+
+    Args:
+        context (OpExecutionContext): The execution context.
+        channel (str): The channel to send the message to.
+        message (str): The message to send.
+    """
+    if os.getenv("SLACK_TOKEN"):
+        context.resources.slack.get_client().chat_postMessage(channel=channel, text=message)
```

Comment:
I used to work on a slack integration and the blast from the past that is `chat_postMessage` hits so hard